.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,61 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Was this file accidentally committed? It doesn't seem necessary for everyone else.